### ISSUE TITLE
[BUG]: Appimage does not work on Ubuntu Bionic

### ISSUE THREAD

**Issue by electrofloat** [martinrotter/rssguard#791]
### Brief description of the issue

./rssguard-4.2.4-1f6d7c0b-linux64.AppImage 
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6Core.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6WebEngineCore.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libglib-2.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstaudio-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstvideo-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstreamer-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libnspr4.so)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgsttag-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)

### How to reproduce the bug?

./rssguard-4.2.4-1f6d7c0b-linux64.AppImage 
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6Core.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6WebEngineCore.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libglib-2.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstaudio-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstvideo-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstreamer-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libnspr4.so)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgsttag-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)

### What was the expected result?

?

### What actually happened?

./rssguard-4.2.4-1f6d7c0b-linux64.AppImage 
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6Core.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6WebEngineCore.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libglib-2.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstaudio-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstvideo-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstreamer-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libnspr4.so)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgsttag-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)

### Debug log

./rssguard-4.2.4-1f6d7c0b-linux64.AppImage 
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6Core.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libQt6WebEngineCore.so.6)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libglib-2.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstaudio-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstvideo-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgstreamer-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libnspr4.so)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libgsttag-1.0.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libsystemd.so.0)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)
/tmp/.mount_rssgualweSWh/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_rssgualweSWh/usr/bin/../lib/libudev.so.1)

### Operating system and version

* OS: Ubuntu Bionic
* RSS Guard version: 4.2.4-1f6d7c0b-linux64.AppImage

**Comment by electrofloat** [comment]
Sorry, I'm dumb. The above is the output on Ubuntu Bionic.

On Ubuntu Jammy, I had to  install libfuse2 then it worked.

**Comment by martinrotter** [comment]
Yes.

Let me clarify. For many years RSS Guard used Ubuntu 18 + Qt 5.x to build the AppImages. Sadly, Qt 5.x is just now old and I decided we need to move on to Qt 6 on all platforms. Due to some bugs in early Qt 6 versions, RSS Guard needs Qt 6.3.0 at minimum to function properly.

Sadly official Qt 6.3.x binaries need Glibc 2.31 or newer to run and Ubuntu 18 does have OLDER Glibc. Therefore OFFICIAL binaries of Qt 6.3.x do not support Ubuntu 18.

https://doc.qt.io/qt-6/supported-platforms.html#linux-x11

Therefore, I had to switch to Ubuntu 20 as AppImage building distro. There is some complex thinking behind this.

So this is not a bug actually. I will update readme and docs and add minimum glibc requirement to it. Sorry for any problems this caused to you.
